pages/EntryListPage/index.js

```diff
@@ -0,0 +1,14 @@
+import JournalEntryList from "@/components/JournalEntryList";
+import Link from "next/link";
+
+export default function EntryListPage() {
+  return (
+    <>
+      {" "}
+      <Link href="/">
+        <p>Back</p>
+      </Link>
```

Comment:
Hi Iris, do you need the <p> tag here? In my opinion, you can just write "Back" in between the <Link> Tags.